rexfw/samplers/rwmc.py

```diff
@@ -8,7 +8,7 @@
 from rexfw.samplers import AbstractSampler
 
 
-RWMCSampleStats = namedtuple('RWMCSampleStats', 'accepted total stepsize')
+RWMCSampleStats = namedtuple('RWMCSampleStats', 'accepted total stepsize neg_log_prob')
```

Comment:
Is this missing a `', '` or are they supposed to be space delimited?

Reply:
They are supposed to be space-delimited. I also think it looks really odd, but well... :thinking:
Hmm I just looked it up in the Python docs:
> The field_names are a sequence of strings such as ['x', 'y']. Alternatively, field_names can be a single string with each fieldname separated by whitespace and/or commas, for example 'x y' or 'x, y'.

What happened to
> There should be one-- and preferably only one --obvious way to do it.

from the Zen of Python?